From c27d3e7349aef6af438844c440e502f9438a201b Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 9 Dec 2021 16:13:50 -0800
Subject: [PATCH] pki files are moved under datadir

---
 src/fu-engine.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- fwupd-1.8.4/src/fu-engine.c~	2022-08-30 09:37:36.000000000 +0000
+++ fwupd-1.8.4/src/fu-engine.c	2022-09-20 14:05:00.656219194 +0000
@@ -8100,9 +8100,10 @@
 	keyring_path = fu_path_from_kind(FU_PATH_KIND_LOCALSTATEDIR_PKG);
 	jcat_context_set_keyring_path(self->jcat_context, keyring_path);
 	sysconfdir = fu_path_from_kind(FU_PATH_KIND_SYSCONFDIR);
-	pkidir_fw = g_build_filename(sysconfdir, "pki", "fwupd", NULL);
+	char *datadir = fu_path_from_kind(FU_PATH_KIND_DATADIR_PKG);
+	pkidir_fw = g_build_filename(datadir, "pki", "fwupd", NULL);
 	jcat_context_add_public_keys(self->jcat_context, pkidir_fw);
-	pkidir_md = g_build_filename(sysconfdir, "pki", "fwupd-metadata", NULL);
+	pkidir_md = g_build_filename(datadir, "pki", "fwupd-metadata", NULL);
 	jcat_context_add_public_keys(self->jcat_context, pkidir_md);
 
 	/* add some runtime versions of things the daemon depends on */
