--- fwupd-1.3.3/plugins/optionrom/fu-optionrom-device.c~	2019-11-01 15:06:16.000000000 +0000
+++ fwupd-1.3.3/plugins/optionrom/fu-optionrom-device.c	2019-11-02 14:00:25.859657185 +0000
@@ -47,6 +47,8 @@
 	g_autoptr(GBytes) fw = NULL;
 	g_autoptr(GFile) file = NULL;
 
+	return NULL;
+
 	/* open the file */
 	rom_fn = g_build_filename (fu_udev_device_get_sysfs_path (udev_device), "rom", NULL);
 	if (rom_fn == NULL) {
